app/components/sidebars/main/channels_list/list/list.js

```diff
@@ -110,7 +114,7 @@ export default class List extends PureComponent {
         this.listRef = ref;
     }
 
-    getSectionConfigByType = (props, sectionType) => {
+    getSectionConfigByType = (props, sectionType, displayName = 'x') => {
```

Comment:
'x' should just be blank I think, or perhaps pass down the display name to the renderItem.